hack/build.sh.d/kn-plugin-workflow.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+readonly quarkus_version="2.13.7.SP1-redhat-00001"
+readonly quarkus_platform_group_id="com.redhat.quarkus.platform"
+readonly dev_mode_image="quay.io/kiegroup/kogito-swf-devmode:1.40"
```

Comment:
We can follow the same pattern as `event` plugin for example? To make this image configurable in later stages of build process. 

https://github.com/openshift-knative/client/blob/release-v1.8/hack/build.sh.d/kn-plugin-event.sh#L8-L9

Reply:
Applicable to `quarkus_version` as well.